src/components/pageheader/PageHeader.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { Typography } from 'antd';
```

Comment:
Sorry, what's this for? Does this optimize the page header for mobile?

Reply:
Yeah, there's a margin above the page header that looks excessive on mobile but fine on all the other window types.